### ISSUE TITLE
Win x64  VS 15 2017, vcpkg, CMake 3.14 Builds has Linker error

### ISSUE THREAD

**Issue by ManuelKugelmann** [GENIVI/vsomeip#78]
If I disable the emptying of the Boost_LIBRARIES variable it just works...
```
    #set(Boost_LIBRARIES "")
    link_directories(${Boost_LIBRARY_DIR_DEBUG})
```
What is the reason for emptying that variable?

**Comment by energystoryhhl** [comment]
boost1.65 msvc14.2 use cmake compile is ok

**Comment by TheDraguun** [comment]
cmake compile might not trigger the code block containing the offending line.

What interests me most, is the reason for emptying that variable....

**Comment by energystoryhhl** [comment]
you can add boost lib and include dir to system path,the cmake will detecte it

**Comment by TheDraguun** [comment]
Thanks for your input, but I have no problem building vsomeip - I just have to disable this in the build script: `set(Boost_LIBRARIES "")` 
I am wondering if maybe there is something wrong with the build script or if I it intentionally is targeting a very specific build environment.

**Comment by mahaase** [comment]
same issue here. i am also just wondering about this line...
IMO that should never a good idea to manipulate the boost variables...

**Comment by lutzbichler** [comment]
Fixed in v3.1.20.1
